packages/picasso/src/Link/Link.tsx

```diff
@@ -51,6 +51,8 @@ export type Props = BaseProps &
     tabIndex?: number
     /** Indicates that the user cannot interact with the Link or its children */
     disabled?: boolean
+    /** Indicates that the link was opened before within this browser session */
+    visited?: boolean
```

Comment:
```suggestion
    visited?: boolean
```
`visited` is only for test purposes, when you add the comment, the documentation is automatically generated in the storybook for this prop.

Reply:
Ah, thanks for the suggestion I was not aware of that. Let me remove the comment then